lib/api/apiconfig.go

```diff
@@ -25,6 +26,7 @@ type DatastoreType string
 const (
 	EtcdV2     DatastoreType = "etcdv2"
 	Kubernetes DatastoreType = "kubernetes"
+	Consul     DatastoreType = "consul"
```

Comment:
Do you think we should call this consulv1 for potential forward compatibility?  I'm not a Consul expert, so I don't know how likely it is that there'd be an incompatible consulv2 in the near future.